examples/pennylane_run_doubly_stochastic.py

```diff
@@ -179,8 +184,8 @@ def circuit(params):
 
 plt.style.use("seaborn")
 plt.plot(cost_GD[:100], label="Vanilla gradient descent")
-plt.plot(cost_SGD100[:100], label="Stochastic gradient descent, $shots=100$")
-plt.plot(cost_SGD1[:100], ".", label="Stochastic gradient descent, $shots=1$")
+plt.plot(cost_SGD100[:100], label="QSGD ($shots=100$)")
+plt.plot(cost_SGD1[:100], ".", label="QSGD ($shots=1$)")

```

Comment:
Can you give an updated screenshot of the rendered tutorial? I suppose that `$shots$` will render in a weird way because it will (as far as I anticipate) be in all italics. better use `QSGD (100 shots)` here.